create_jenkins_job.py

```diff
@@ -191,15 +191,15 @@ def create_job(os_name, job_name, template_file, additional_dict):
                 'cmake_build_type': 'Debug',
                 'compile_with_clang_default': 'true',
                 'time_trigger_spec': PERIODIC_JOB_SPEC,
-                'mailer_recipients': DEFAULT_MAIL_RECIPIENTS,
+                'mailer_recipients': DEFAULT_MAIL_RECIPIENTS + ' amazon-b9-ros@amazon.com',
```

Comment:
ros-contributions@amazon.com would be better IMHO - is this email used somewhere else?

Reply:
> is this email used somewhere else?

I pulled it off the [rcl_logging](https://github.com/ros2/rcl_logging/blob/efeebc970934ec0f9c7f64d0c175345074fa8ef7/rcl_logging_noop/package.xml#L8) maintainer entry.